packages/nextjs/app/builders/0x2723A2756ecb99b3B50f239782876fB595728AC0/page.tsx

```diff
@@ -0,0 +1,18 @@
+"use client";
+
+import type { NextPage } from "next";
+
+const PersonalPage: NextPage = () => {
+  return (
+    <div>
+      <iframe style={container} src="https://arjun.wtf"></iframe>
```

Comment:
Lol this is nice approach and completely agree that intention might be to streamline the process or include dynamic content. 

But it also exposes some security issues, could you add all the content in this repo file itself, maybe you could keep it simple and also add link to your this portfolio itself similar to other PR's ? 
This ensures that all content is reliably available and displayed as intended, without dependency on external sites which we cannot control 

Thanks @ARJUN-R34, I hope you understand 🙌

Reply:
@technophile-04 
Yes that makes sense. Will update it.

Reply:
Hello @technophile-04, I have updated the code as per your instructions.